doc/modules/model_evaluation.rst

```diff
@@ -939,9 +939,9 @@ To make this more explicit, consider the following notation:
 * :math:`y_l` the subset of :math:`y` with label :math:`l`
 * similarly, :math:`\hat{y}_s` and :math:`\hat{y}_l` are subsets of
   :math:`\hat{y}`
-* :math:`P(A, B) := \frac{\left| A \cap B \right|}{\left|\right|}` for some
+* :math:`P(A, B) := \frac{\left| A \cap B \right|}{\left|A\right|}` for some
   sets :math:`A` and :math:`B`
-* :math:`R(A, B) := \frac{\left| A \cap B \right|}{\left|A\right|}`
+* :math:`R(A, B) := \frac{\left| A \cap B \right|}{\left|B\right|}`
```

Comment:
Recently, the definition for `y_hat` and `y` was flipped:

https://github.com/scikit-learn/scikit-learn/blob/71d1e849557a8ee05204711ddfcfc890d694b889/doc/modules/model_evaluation.rst?plain=1#L933-L934

This means `A` on the bottom is correct, because we use `R(y, y_hat)`.